lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -41,7 +48,7 @@ def resolve_stickiness(stickiness, context)
         when 'random'
           random
         when 'default'
-          context.user_id || context.session_id || random
+          context&.user_id || context&.session_id || random
```

Comment:
note that in the special case of `stickiness => default`, it will still crash if you send a `context` of type `String` for example. But if you are doing that, you are really asking for trouble.